src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1149,6 +1145,33 @@ namespace cryptonote
       END_KV_SERIALIZE_MAP()
     };
   };
+
+  struct COMMAND_RPC_GENERATEBLOCKS
+  {
+    struct request
+    {
+      uint64_t amount_of_blocks;
+      uint64_t reserve_size;
```

Comment:
That is pointless since it's not used. This is used by pools to embed something miner specific, but here the blocks are generated without adding anything in the reserved space AFAICT.

Reply:
ack